incubator/cloudbeaver/templates/statefulsets.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "cloudbeaver.fullname" . }}
+spec:
+  serviceName: cloudbeaver
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloudbeaver
+  template:
+    metadata:
+      labels:
+        app: cloudbeaver
```

Comment:
I'm not sure if we want multiple installations of this chart. If we do, then we should set the `app` to `{{ include "cloudbeaver.fullname" . }}` instead.

Reply:
Done!